### PR TITLE
add datetime to posts

### PR DESCRIPTION
**Pull request by lonniebe**

### CODE REVIEW

File: BE/lib/rest/profile.py
Comment (pazbechor) [inline BE/lib/rest/profile.py:1]: You can delete it

File: BE/lib/bussiness_logic/posts_manager.py [hunk 1]
```diff
@@ -37,14 +38,15 @@ def create_new_post(self, user: User, new_post: NewPost) -> PostOut:
...
-            Post.post_id.name: str(uuid.uuid4())
+            Post.post_id.name: str(uuid.uuid4()),
         }
         post_to_add_dict.update(new_post.dict())
 
         self.db_session.add(Post(**post_to_add_dict))
         self.db_session.commit()
         # Create response
         post_to_add_dict["author"] = self.db_session.get_user_query(user.user_email).first().dict()
+        post_to_add_dict["published_time"] = datetime.now()
```
Comment (pazbechor) [inline BE/lib/bussiness_logic/posts_manager.py:49]: Why did you add it after adding to DB ?

Comment (pazbechor) [reply BE/lib/bussiness_logic/posts_manager.py:49]: I think you also can query the just added post from DB and don't do it here.

File: BE/lib/utils/db/models/companies.py [hunk 2]
```diff
@@ -6,6 +6,8 @@
 class Company(Base):
     __tablename__ = "company"
 
-
     id = Column(String, primary_key=True, unique=True, index=True)
+    name = Column(String, index=True, nullable=False)
```
Comment (pazbechor) [inline BE/lib/utils/db/models/companies.py:10]: It's not related to this PR

File: BE/lib/rest/profile.py
Comment (lonniebe) [reply BE/lib/rest/profile.py:1]: done.

File: BE/lib/bussiness_logic/posts_manager.py (hunk 1 above)
Comment (lonniebe) [reply BE/lib/bussiness_logic/posts_manager.py:49]: because if i add it before, i can't add it to the db

File: BE/lib/utils/db/models/companies.py (hunk 2 above)
Comment (lonniebe) [reply BE/lib/utils/db/models/companies.py:10]: removed.

**Review by pazbechor — APPROVED** [review]
